qrcode/qrcode.go

```diff
@@ -100,7 +103,10 @@ func Reader(s string) (res string) {
 	if s == "" {
 		res, _ = FromScreenshot()
 	} else {
-		r, _ := Scan(s)
+		r, err := Scan(s)
+		if err != nil {
+			fmt.Printf("warning: %s\n", err.Error())
```

Comment:
I'm not sure if this function should be silent at all costs

Reply:
I think should be good - this should be displayed at the user calling the registration code. We just have to be sure (generally speaking) that we don't break printing the QRcode